### DISCUSSION TITLE
Parsing function for the actual 'value' set from the datepicker

### DISCUSSION THREAD

**Discussion by Huiet** [mantinedev/mantine#2724] [Q&A]
Is there currently a way to handle parsing of dates (parsing as in modifying what is stored in the form value) to be something other than a date object, or is that not even feasible?  

In my specific use case, we want dates to be in the MM/DD/YYYY format, and in an ideal world I'd be able to set up a parsing function directly on the datePicker component (similar to what exists on numberInput) to be able to have the actual value stored as a MM/DD/YYYY format as the user interacts with it (via free input or date picking via calendar).

My gut tells me maybe not, but a man can dream

**Comment by rtivital** [comment]
You can format the date object you receive from the component according to your requirements. There is no way of supporting all existing string formats.
